### DISCUSSION TITLE
Markdown to PDF - Table columns overlap

### DISCUSSION THREAD

**Discussion by marek2356** [jgm/pandoc#10195] [Q&A] [answered]
Hi,

I have a Markdown file with text, and tables that I would like to convert to PDF. But in one of the tables the columns overlap. How can I fix this issue?

This is a MWE with the overlapping table:

Markdown file: 
[report_giskard.md](https://github.com/user-attachments/files/17041949/report_giskard.md)

Content of the markdown file:
```
|    | Vulnerability   | Level   | Metric            | Deviation                                                                                   | Description                                                                                                                                                             |
|---:|:----------------|:--------|:------------------|:--------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
|  1 | Robustness      | major   | Fail rate = 1.000 | Adding special chars `\r` in `question` can make the model to produce unexpected outputs.   | Injecting long sequences of control characters `\r` in the value of `question` can alter the model's output significantly, producing unexpected or off-topic outputs.   |
|  2 | Robustness      | major   | Fail rate = 1.000 | Adding special chars `\x08` in `question` can make the model to produce unexpected outputs. | Injecting long sequences of control characters `\x08` in the value of `question` can alter the model's output significantly, producing unexpected or off-topic outputs. |
```

Command used for convertion:
` pandoc report_giskard.md -o demo.pdf`

PDF output:
[demo.pdf](https://github.com/user-attachments/files/17041965/demo.pdf)

OS: Windows 10

`pdflatex --version` outputs:
```
pdfTeX 3.141592653-2.6-1.40.26 (TeX Live 2024)
kpathsea version 6.4.0
Copyright 2024 Han The Thanh (pdfTeX) et al.
There is NO warranty.  Redistribution of this software is
covered by the terms of both the pdfTeX copyright and
the Lesser GNU General Public License.
For more information about these matters, see the file
named COPYING and the pdfTeX source.
Primary author of pdfTeX: Han The Thanh (pdfTeX) et al.
Compiled with libpng 1.6.43; using libpng 1.6.43
Compiled with zlib 1.3.1; using zlib 1.3.1
Compiled with xpdf version 4.04
```

Thank you.

**Comment by jgm** [comment, marked as answer]
Pandoc parses the table with relative column widths that are based on the relative widths in the source.  (These widths are needed for LaTeX output, as LaTeX doesn't compute them automatically as browsers do.)

In your table, the second column is  apparently not wide enough for its content.  So, you can try making it wider.  You would do this by adjusting the line with the `----` so that it is longer in the second column. 

For example, setting it to
```
|-:|:------------|:-------|:-------|:---------------------|:------------------|
```
will give reasonable results.  (Note that unless this line is shorter than the 78 characters, absolute widths of the lines don't matter, only relative ones. `pandoc -t native` will show you how it is parsed and what the relative widths are.)
